jobs/ci-run/functional/proving-grounds-functional-tests.yml

```diff
@@ -22,20 +22,22 @@
       - multijob:
           name: ProvingGroundsFunctionalTestsAmd64
           projects:
-            - name: nw-deploy-bionic-amd64-equinix
+            - name: nw-deploy-jammy-amd64-equinix
               current-parameters: true
               enable-condition: '!["2.8"].contains("${JUJU_VERSION_MAJOR_MINOR}")' # Provider support is only available on 2.9+
-            - name: nw-deploy-bionic-amd64-lxd
+            - name: nw-deploy-focal-amd64-lxd
+              current-parameters: true
+            - name: nw-deploy-jammy-amd64-lxd
               current-parameters: true
             - name: nw-deploy-bionic-eks
               current-parameters: true
             - name: nw-model-migration-amd64-vsphere
               current-parameters: true
-            - name: nw-deploy-bionic-vsphere
+            - name: nw-deploy-jammy-vsphere
```

Comment:
no need for series in the job name, we don't care about that here
it's just a deploy to vsphere test